tests/data/test_triplet.py

```diff
@@ -0,0 +1,50 @@
+from iohub import open_ome_zarr
+from pytest import mark
+
+from viscy.data.triplet import TripletDataModule
+
+
+@mark.parametrize("include_wells", [None, ["A/1", "B/2"]])
+def test_datamodule_setup_fit(
+    preprocessed_hcs_dataset, tracks_hcs_dataset, include_wells
+):
```

Comment:
The current test does not test for the exclusion of the fovs. Can we add an extra fov `/C/3` to the hcs plate and exclude this position? and for the `/B/3` exclude some of the position?